### PR TITLE
Update README.md :Added TensorFlow: Advanced Techniques Specialization to resources tab

### PR DESCRIPTION
**Pull request by AdityaKane2001**
Added TensorFlow: Advanced Techniques Specialization to resources tab. Extremely helpful specialization by Laurence Moroney for more in depth knowledge of TensorFlow.

### CODE REVIEW

**Comment by AdityaKane2001** [comment]
@googlebot I signed it!

File: tensorflow/python/keras/engine/base_layer.py [hunk 1]
```diff
@@ -2152,7 +2152,7 @@ def output(self):
     """Retrieves the output tensor(s) of a layer.
 
     Only applicable if the layer has exactly one output,
-    i.e. if it is connected to one incoming layer.
+    i.e. if it is connected to one outgoing layer.
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/engine/base_layer.py:2155]: I am not sure this is right

Comment (AdityaKane2001) [reply tensorflow/python/keras/engine/base_layer.py:2155]: Sorry for inconvenience, I realised that but forgot to commit the same. The change is incorrect, please revert.

**Review by AdityaKane2001 — COMMENTED** [review]
Thanks for the correction

File: tensorflow/python/keras/engine/base_layer.py (hunk 1 above)
Comment (mihaimaruseac) [reply tensorflow/python/keras/engine/base_layer.py:2155]: You have to revert it by pushing a new commit.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by AdityaKane2001** [comment]
Requested changes done.

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**